fe/fe-core/src/main/java/org/apache/doris/persist/CleanQueryStatsInfo.java

```diff
@@ -69,4 +77,8 @@ public Scope getScope() {
     public void write(DataOutput out) throws IOException {
         Text.writeString(out, GsonUtils.GSON.toJson(this));
     }
+
+    private Scope translateToLegacyScope(CleanQueryStatsCommand.Scope scope) {
```

Comment:
move this method into CleanQueryStatsCommand class, then this file can be unchanged